.envrc

```diff
@@ -0,0 +1,2 @@
+export PREFAB_INTEGRATION_TEST_API_KEY="306-test-P124-E143-SDK-61ffa8e9-da02-49fe-8f0e-f8780325b39e"
```

Comment:
Did you mean to check this in? Should we just hardcode it in `IntegrationTest.java` if so?